cmd/ls.go

```diff
@@ -42,14 +42,14 @@ func printFolderMetadata(w io.Writer, e *files.FolderMetadata, longFormat bool)
 	if longFormat {
 		fmt.Fprintf(w, "-\t-\t-\t")
 	}
-	fmt.Fprintf(w, "%s\n", e.Name)
+	fmt.Fprintf(w, "%s\t\t", e.Name)
```

Comment:
@diwakergupta Yea, you're right. I'm going to make a less trivial solution later today. It will have to involve using the `exec.Command("stty", "size")` to actually ge the size of the terminal and print accordingly